src/SFA.DAS.EmployerAccounts/Queries/GetAccountLegalEntityRemove/GetAccountLegalEntityRemoveQueryHandler.cs

```diff
@@ -84,6 +84,7 @@ private async Task<bool> SetRemovedStatusBasedOnCommitments(long accountId, Acco
 
             return commitmentConnectedToEntity == null || (commitmentConnectedToEntity.ActiveCount +
                                                            commitmentConnectedToEntity.PendingApprovalCount +
+                                                           commitmentConnectedToEntity.CompletedCount +
```

Comment:
I think you need to use WithdrawnCount to include Stopped apprenticeships. Withdrawn = Stopped. Although completed is probably also a valid check.

Reply:
Fixed